UIInfoSuite2/Infrastucture/Tools.cs

```diff
@@ -114,5 +114,41 @@ public static Item GetHoveredItem()
 
             return hoverItem;
         }
+
+        public static void GetSubTexture(Color[] output, Color[] originalColors, Rectangle sourceBounds, Rectangle clipArea)
+        {
+            //Create output array
```

Comment:
I think this comment is quite misleading and should be deleted.

Reply:
yeah it slipped through from when I was writing the method in a different way, I'll get rid of all the comments